### PR TITLE
LiftIO FTW

### PR DESCRIPTION
**Pull request by ArturGajowy**
## Overview
<sup>_What this PR does, and why it's needed_</sup>

bors try

### JIRA ticket:
<sup>_Create it if there isn't one already._</sup>

### Notes
<sup>_Optional. Add any notes on caveats, approaches you tried that didn't work, or anything else._</sup>

### Please make sure that this PR:
- [x] is at most 200 lines of code (excluding tests),
- [x] meets [RChain development coding standards](https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards),
- [x] includes tests for all added features,
- [x] has a reviewer assigned,
- [x] has [all commits signed](https://rchain.atlassian.net/wiki/spaces/DOC/pages/498630673/How+to+sign+commits+to+rchain+rchain).

### [Bors](https://bors.tech/) cheat-sheet:

- `bors r+` runs integration tests and merges the PR (if it's approved),
- `bors try` runs integration tests for the PR,
- `bors delegate+` enables non-maintainer PR authors to run the above.

### CODE REVIEW

**Comment by ArturGajowy** [comment]
bors try

**Comment by ArturGajowy** [comment]
bors try

**Comment by ArturGajowy** [comment]
bors try

File: casper/src/main/scala/coop/rchain/casper/util/rholang/InterpreterUtil.scala [hunk 1]
```diff
@@ -32,7 +31,7 @@ object InterpreterUtil {
 
   //Returns (None, checkpoints) if the block's tuplespace hash
   //does not match the computed hash based on the deploys
-  def validateBlockCheckpoint[F[_]: Sync: Log: BlockStore: ToAbstractContext](
+  def validateBlockCheckpoint[F[_]: Monad: Log: BlockStore: Sync: LiftIO](
```
Comment (lukasz-golebiewski) [inline casper/src/main/scala/coop/rchain/casper/util/rholang/InterpreterUtil.scala]: Isn't `Sync` a `Monad` already?

Comment (ArturGajowy) [reply casper/src/main/scala/coop/rchain/casper/util/rholang/InterpreterUtil.scala]: Sure it is :D :+1:

**Comment by ArturGajowy** [comment]
bors try

**Comment by ArturGajowy** [comment]
bors try

**Comment by ArturGajowy** [comment]
bors try

**Pull request closed without merging**